src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => <>Put the user here</>;
+export const UserInfo = ({ IdOfUser, Users }) => {
+  const user = Users.find(given => given.id === IdOfUser);
```

Comment:
so here, you can import that array of users and try to find your user, but the best way is to move this logic in PostInfo, 'cause the user can be undefined, and in this case, it is better not to render the UserInfo component